silk/unit_tests/test_log_replay.py

```diff
@@ -50,7 +50,7 @@ def testReplayFormNetwork(self):
         log_path = self.get_log_path("form_network_log.txt")
         with log_path.open() as log:
             for line in log:
-                
+                pass
```

Comment:
Seems missing the test ?

Reply:
This was a placeholder in the older commits.